content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -388,26 +236,28 @@ sleep.legacy to httpbin.bar: 000
 command terminated with exit code 56
 {{< /text >}}
 
-If we have more services in namespace `bar`, we should see traffic to them won't be affected. Instead of adding more services to demonstrate this behavior,
-we edit the policy slightly to apply on a specific port:
+To refine the mTLS settings per port, you must configure the `portLevelMtls` section. For example, the following peer authentication policy requires mTLS on all ports, except port `80`:
 
 {{< text bash >}}
 $ cat <<EOF | kubectl apply -n bar -f -
-apiVersion: "authentication.istio.io/v1alpha1"
-kind: "Policy"
+apiVersion: "security.istio.io/v1beta1"
+kind: "PeerAuthentication"
 metadata:
   name: "httpbin"
+  namespace: "bar"
 spec:
-  targets:
-  - name: httpbin
-    ports:
-    - number: 1234
-  peers:
-  - mtls: {}
+  selector:
+    matchLabels:
+      app: httpbin
+  mtls:
+    mode: STRICT
+  portLevelMtls:
+    80:
+      mode: DISABLE
 EOF
 {{< /text >}}
 
-And a corresponding change to the destination rule:
+As before, you also need a destination rule, for example:
```

Comment:
i think `for example` here is a bit redundant. you need to have DR exactly as below. IIUC